stable/datadog/templates/daemonset.yaml

```
@@ -274,6 +282,14 @@ spec:
             path: {{ default "/var/lib/datadog-agent/logs" .Values.datadog.logsPointerHostPath | quote }}
           name: pointerdir
         {{- end }}
+        {{- if (and (.Values.datadog.logsEnabled) (.Values.datadog.logsConfigContainerCollectAll)) }}
+        - hostPath:
+            path: /var/log/pods
+          name: logpodpath
+        - hostPath:
+            path: /var/lib/docker/containers
```

docker is not the only one container runtime possibly used with this chart; so, it can be nice to offer the possibility to overwrite this path.

```suggestion
            {{- if .Values.datadog.containerLogsPath }}
            path: {{ .Values.datadog.containerLogsPath | quote }}
            {{- else }}
            path: /var/lib/docker/containers
            {{- end }}
```